test/invoice_item_test.rb

```diff
@@ -6,8 +6,8 @@ class InvoiceItemTest < MiniTest::Test
   attr_reader :parent, :invoice_item
 
   def setup
-    sales_engine = SalesEngine.new
-    @parent = InvoiceItemRepository.new('test/support/invoice_items.csv', sales_engine)
+    sales_engine = SalesEngine.new('test/support')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.